Tasks/Maven/maventask.ts

```diff
@@ -203,6 +203,55 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
     return mvnsq;
 }
 
+function processMavenOutput(data) {
+    data = data.toString();
+    var input = data;
+    var severity = 'NONE';
+    if(data.substring(0, 1) == '[') {
```

Comment:
will update but == is used very commonly in all the tasks, I guess since most of the types are string anyway.